apps/web/middleware.ts

```diff
@@ -171,15 +171,11 @@ export const config = {
     "/api/auth/signup",
     "/api/trpc/:path*",
     "/login",
-    "/auth/:path*",
```

Comment:
/auth/login has been here regardless of app router vs pages router

but other auth subpaths should be here only after being migrated to app router